app/src/relauncher.js

```diff
@@ -0,0 +1,25 @@
+const electron = require('electron');
+const log = require('electron-log');
+
+const appEnv = require('./appenv.js');
+
+/**
+ * The electron app.
+ */
+const app = (electron.app || electron.remote.app);
```

Comment:
Given relaunch doesn't work properly in dev and `isDev` won't be correct outside the main process, I don't think we should try using `electron.remote.app` here.

Reply:
Just did the below change.